tests/js/plugin/enhancedanchoring-test.coffee

```diff
@@ -223,14 +228,13 @@ describe 'Annotator.Plugin.EnhancedAnchoring', ->
       assert.include hls, anchor2.highlight[anchor2.startPage]
       assert.notInclude hls, anchor3.highlight[anchor3.startPage]
 
-  describe 'Anchor.scrollIntoView()', ->
+  describe 'Anchor.scrollToView()', ->
     it 'calls scrollIntoView() on the highlight', ->
```

Comment:
This isn't always the case, though, right? It should call it if it's real, otherwise it scrolls to the adjacent page and then the scrolling to the highlight happens asynchronously, correct?